src/main/kotlin/com/github/devcordde/devcordbot/commands/general/TagCommand.kt

```diff
@@ -405,7 +405,7 @@ class TagCommand : AbstractRootCommand() {
                 MarkdownSanitizer.escape(tag.content).replace("\\```", "\\`\\`\\`") // Discords markdown renderer suxx
             if (content.length > MAX_CONTENT_LENGTH) {
                 val message = context.respond(Emotes.LOADING)
-                val code = HastebinUtil.postToHastebin(content, context.bot.httpClient)
+                val code = HastebinUtil.postToHastebin(tag.content, context.bot.httpClient)
```

Comment:
Don't escape the markdown if you don't use it

Reply:
But we need its length for the check and it shouldn't really matter performance wise.

Reply:
Ofc my bad